components/MVP/CovidSiteAlert/CovidSiteAlert.module.scss

```diff
@@ -0,0 +1,19 @@
+@import 'styles/uswdsDependencies';
+@import 'styles/mvp/mixins';
+
+.usa-site-alert--info {
```

Comment:
In order to not confuse local CSS module class names with global/USWDS classnames, I'd suggest using a different naming convention, maybe `.covidSiteAlert`?